Content.IntegrationTests/Tests/Weapons/WeaponSniperTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Content.IntegrationTests.Tests.Interaction;
+using Content.Shared.Damage;
+using Content.Shared.Weapons.Ranged.Components;
+using Content.Shared.Weapons.Ranged.Events;
+using Content.Shared.Wieldable.Components;
+using Robust.Shared.Prototypes;
+
+namespace Content.IntegrationTests.Tests.Weapons;
+
+public sealed class WeaponSniperTests : InteractionTest
+{
+    private static readonly EntProtoId MobHuman = "MobHuman";
+    private static readonly EntProtoId SniperMosin = "WeaponSniperMosin";
+
+    [Test]
+    public async Task SniperMosinInteractionTest()
+    {
+        var urist = await SpawnTarget(MobHuman);
+        var damageComp = Comp<DamageableComponent>(urist);
+        var startDamage = damageComp.TotalDamage.Value;
```

Comment:
```suggestion
        // The spawned mob starts with some airloss damage
        var startDamage = damageComp.TotalDamage.Value;
```
Was that the reason for this? If yes, leave it as a code comment.